packages/stream/src/index.ts

```diff
@@ -70,7 +79,15 @@ export const createStream = (streamSource: StreamSourceDescription): StreamRetur
   );
 
   onCleanup(() => stop(stream()));
-  return [stream, { mutate, refetch, stop: () => stop(stream()) }];
+  return [
+    stream,
+    {
+      mutate,
+      refetch,
+      mute: (muted?: boolean) => mute(stream(), muted),
```

Comment:
Calling `mute()` will track `stream`. Is that desirable?
Same with `stop()` it seems.
The rest looks 👍

Reply:
In most cases, these will be called inside events, which are outside of the reactive scope. I hadn't considered other cases yet.

Reply:
Sure, you can leave it if you think that's fine. But I would wrap it with `untrack` to avoid some weird edge-cases. I think it would be a hard issue to track down.

Reply:
You're right, and untrack is cheap. I'll add it once I get to it. Too tired today.